### DISCUSSION TITLE
플리퀘 검토중 봇이 클로즈

### DISCUSSION THREAD

**Discussion by n47turbo** [devrel-kr/HackaLearn#424] [1 Q&A - HackaLearn 운영]
깃허브 레포 검토중에
(PR은 오전 12시 전 제출)
봇이 자동으로 클로즈 해버렸습니다

이에대한 경우는 완주처리가 안되는건지요?
어처구니 없게 클로즈 되버린지라 정확한 답변을 받고싶습니다
감사합니다..

**Comment by justinyoo** [comment]
@kimka2013 8월 16일 0시 전에 생성한 PR에 대해서는 12시간의 PR 수정 기간을 드렸구요, 8월 16일 12시를 넘어서까지 수정 요청이 반영되지 않은 PR들은 모두 자동으로 닫혔습니다. 따라서 해당 부분에 대해서는 안타깝지만 반영되지 않습니다.

**Reply by n47turbo** [reply]
@justinyoo 그러하면 심사대상도 아닌게 되버린건가요?

**Reply by justinyoo** [reply]
모든 챌린지를 완주한 팀만을 대상으로 심사합니다.
